u2f-ref-code/java/src/com/google/u2f/key/messages/TransferAccessMessage.java

```diff
@@ -102,6 +105,40 @@ public static TransferAccessMessage fromBytes(byte[] data) throws U2FException {
   }
   
   /**
+   * Write a TransferAccessMessage into a raw byte array.
+   * @throws U2FException 
+   */
+  public byte[] toBytes() throws U2FException {
+    int sizeOfSequenceNumber = 1;
+    int lengthField_signatureUsingAuthenticationKey = 1;
```

Comment:
don't use underscores and camel case.

use came case for member variable names.

Also, it's sizeOfLengthField, right?  "sizeof" is important I think